src/concuerror_callback.erl

```diff
@@ -199,7 +199,13 @@ instrumented_call(Module, Name, Arity, Args, _Location,
       [Term] = Args,
       try
         Symbol = ets:lookup_element(Processes, Term, ?process_symbolic),
-        {{didit, Symbol}, Info}
+        PName = ets:lookup_element(Processes, Term, ?process_name),
+        Pretty = case PName =:= ?process_name_none of
+                   true -> Symbol;
+                   false -> lists:flatten(
+                              io_lib:format("<~s/~s>", [Symbol, PName]))
+                 end,
+        {{didit, Pretty}, Info}
```

Comment:
This works for tests where processes are 'deadlocked', bit won't work in general, as the logger accesses this information after an interleaving has been completely explored, at which point some processes will have unregistered their names.